pkg/reloader/reloader.go

```diff
@@ -29,40 +30,42 @@ import (
 // It optionally substitutes environment variables in the configuration.
 // Referenced environment variables must be of the form `$(var)` (not `$var` or `${var}`).
 type Reloader struct {
-	logger          log.Logger
-	reloadURL       *url.URL
-	cfgFile         string
-	cfgEnvsubstFile string
-	ruleDirs        []string
-	ruleInterval    time.Duration
-	retryInterval   time.Duration
+	logger        log.Logger
+	reloadURL     *url.URL
+	cfgFile       string
+	cfgOutputFile string
+	ruleDirs      []string
+	ruleInterval  time.Duration
+	retryInterval time.Duration
 
 	lastCfgHash  []byte
 	lastRuleHash []byte
 }
 
+var magicGzip = []byte{0x1f, 0x8b, 0x08}
```

Comment:
Every magic in code needs to be at least documented (: 

Also those aren't magic, just first gzip bytes, so we can name it properly.. `firstGzipBytes`?